package.json

```diff
@@ -25,7 +25,8 @@
     "react-responsive-carousel": "^3.2.23",
     "recharts": "^2.4.3",
     "suncalc": "^1.9.0",
-    "swr": "^2.0.3"
+    "swr": "^2.0.3",
+    "tailwindcss": "^3.2.7"
```

Comment:
This is there a reason `tailwindcss` was moved from the `devDependencies` section to the `dependencies` section here?

To move back:
`yarn uninstall tailwindcss`
`yarn add -D tailwindcss`